web/index.html

```diff
@@ -1,12 +1,26 @@
 <html>
 	<head>
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
 		<title>My first three.js app</title>
-		<style>
+		<style type="text/css">
 			body { margin: 0; }
 			canvas { width: 100%; height: 100% }
+			#myButtonID {
+				position: fixed;
+			    top: 20px;
+			    right: 20px;
+			    opacity: 0.7;
+			}
 		</style>
 	</head>
 	<body>
+		<div>
+			<button type="button" class="btn btn-secondary" id="myButtonID">
+				<i class="fas fa-rocket"></i>  Submit</button>
```

Comment:
I would fix the indentation here and also change the button id from _Submit_ to _Moon View_ when on the solar system scene and _Planet View_ or something like that when on the moon page. :+1:

Reply:
Good catch! I forgot to change the name. Changed the button to "Fly to Moon" and "Fly Back"